.github/workflows/pre-merge-check.yml

```diff
@@ -73,6 +74,27 @@ jobs:
           echo "Deployment steps will be added in separate tickets:"
           echo "Deploy test harness stack"
           echo "Deploy IPV stub stack"
+
+      - name: SAM Build test harness stack
+        working-directory: ./test-harness
+        run: sam build -t template.yaml
+
+      - name: SAM Deploy F2F test harness stack
```

Comment:
We need the condition in here and for api discussed before so this is not deploying for every dependabot PR. Until we have the delete in place